cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -223,7 +228,8 @@ public void getAllApps(HttpRequest request, HttpResponder responder,
         names.add(name);
       }
     }
-    responder.sendJson(HttpResponseStatus.OK, applicationLifecycleService.getApps(namespace, names, artifactVersion));
+    responder.sendJson(HttpResponseStatus.OK,
```

Comment:
Why make this change instead of letting `AbstractHttpResponder` to convert object to JSON?

Reply:
Because `HttpResponder` no longer has the `sendJson` method as it is error-prone. It requires caller to encode the object explicitly.